sn_node/src/put_validation.rs

```diff
@@ -28,12 +25,6 @@ use std::collections::BTreeSet;
 use tokio::task::JoinSet;
 use xor_name::XorName;
 
-/// The maximum number of double spend attempts to store that we got from PUTs
-const MAX_DOUBLE_SPEND_ATTEMPTS_TO_KEEP_FROM_PUTS: usize = 15;
-
-/// The maximum number of double spend attempts to store inside a record
-const MAX_DOUBLE_SPEND_ATTEMPTS_TO_KEEP_PER_RECORD: usize = 30;
-
```

Comment:
The MAX is now 2 as this measure protected us until the attackers reached the MAX double spent attempt to keep!